deployments/engine/templates/deployment.yaml

```diff
@@ -72,6 +75,8 @@ spec:
           mountPath: /tmp
         - name: tmp
           mountPath: /.ollama
+        readinessProbe:
```

Comment:
I'm curious why a readiness probe with one replica count is used. Is it for detecting errors in somewhere alerting systems and handling failures manually?

Reply:
Yeah that's the main reason. In particular when the initial installation fails, we would like to let end users know.

Also this helps when inference-manager-engine is get deployed. An existing pod is kept until a new one becomes ready.